src/util/orm.py

```diff
@@ -726,10 +726,15 @@ def _add_magic_fields(self):
 
         patches.append(patch.object(BaseModel, "_add_magic_fields", _add_magic_fields))
 
-    if version_gte("saas~16.4"):
-        # 3.5.3 allow loading manual fields
+    # 3.5.3 allow loading manual fields
+
+    if version_between("saas~16.4", "saas~18.4"):
```

Comment:
This includes 18.4, I'd suggest you invert the checks
```
if version >= 18.4:
  ...
elif version >= 16.4:
  ...
```
Note that there are some errors in the CI.

Reply:
OK, we will need some dance to merge the PRs in the right order.
cc: @KangOl

Reply:
community should be deployed first, even if we dont merge a util fix for a while the outcome will be as it is now. But if we merge a patch before the community deployment we will get the error that is currently in the ci

Reply:
Do not forget to put the check for the versions in the right order.
https://github.com/odoo/upgrade-util/blob/5efae69424301785b96c451da7b26132e93f9a51/src/util/misc.py#L123